compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs

```diff
@@ -247,6 +247,13 @@ featureExperimental = Feature
     , featureCppFlag = Just "DAML_EXPERIMENTAL"
     }
 
+-- | CPP flags of past features that have become part of LF but that some
+-- clients might still depend on being defined.
+foreverCppFlags :: [T.Text]
+foreverCppFlags =
```

Comment:
Looks good :) Should we create an error in compiler/damlc/daml-prim-src/DA/Internal/NatSyn.daml if DAML_NAT_SYN is not defined, since it always should be when we build prim from now on?

Reply:
Addressed in https://github.com/digital-asset/daml/pull/18246/commits/23b03eeaea3b501bac4d87cf4f2093437935417e